galaxy/wrapper/computeMatrix.xml

```diff
@@ -25,6 +25,9 @@
             @THREADS@
 
             #if $output.showOutputSettings == "yes"
+                #if $output.saveData:
```

Comment:
@vivekbhr is this a intended modification or a bad merge? I think @dpryan79 removed this in an other PR today.